pkgs/servers/gemini/molly-brown/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, buildGoPackage , fetchgit, nixosTests }:
+
+buildGoPackage  rec {
+  pname = "molly-brown"
+  version = "unstable-2020-07-06";
+  rev = "2e510328ef1737c67641c588095e4628e3dfa8d3";
+
+  goPackagePath = "tildegit.org/solderpunk/molly-brown";
+
+  src = fetchgit {
+    inherit rev;
```

Comment:
I'd put the revision directly here and remove `rec` since it seems to be used here only

Reply:
The go2nix tool placed `rec` here and I don't know why. If it is harmless I would keep it as.